### ISSUE TITLE
Updating library versions isn't consistent

### ISSUE THREAD

**Issue by Versatilus** [chiunhau/p5-manager#34]
When the program is run with the update command it will create a "libraries" folder with the most recent p5 files in the current folder, no matter where it is run it from. It seems to me that it was intended to be run from inside a project's folder or a collection.

I believe the program should be made more context aware when running updates so that it isn't writing blindly to the current folder and also have a flag to globally update the files new projects are drawn from.

**Comment by mehedi432** [comment]
i can not update p5 manager version with p5 update|| p5 u..It says it updated but when i want to know the version it remains the old one...

**Comment by chiunhau** [comment]
@Versatilus @mehedi432 I agree that currently the way p5-manager update its libraries is not well-designed and it need some time/work to rethink/rebuild. For the moment if you face any problem, please just manually download and update the libraries. 👍

**Comment by chiunhau** [comment]
@mehedi432 I think it is due to misunderstanding of the update command (that's my bad). What `p5 update` actually does is: 1. Download the latest p5 libraries release from official p5.js github. 2. Replace the old libraries in your existed collection. 

And `p5 version` only tells the current version of p5-manager (that is, this package), not the p5.js libraries.

**Comment by mehedi432** [comment]
okay

**Comment by mehedi432** [comment]
that means when I command "p5 u" than the library is updated but the p5 manager is in v4....

**Comment by mehedi432** [comment]
that means when I command "p5 u" than the library is updated but the p5 manager is in v4....
